pandas-stubs/core/frame.pyi

```diff
@@ -1683,7 +1686,14 @@ class DataFrame(NDFrame, OpsMixin):
     # def from_dict
     # def from_records
     def ge(self, other, axis: Axis = ..., level: Level | None = ...) -> DataFrame: ...
-    # def get
+    @overload
+    def get(self, key: Hashable, default: None = ...) -> Series | None: ...
+    @overload
+    def get(self, key: Hashable, default: _T = ...) -> Series | _T: ...
```

Comment:
```suggestion
    def get(self, key: Hashable, default: _T ) -> Series | _T: ...
```
I think the `...` should be removed because you would only return `_T` if a specific non-None value was specified.